core/src/main/java/de/danoeh/antennapod/core/glide/FastBlurTransformation.java

```diff
@@ -12,6 +12,7 @@
 import java.security.MessageDigest;
 
 public class FastBlurTransformation extends BitmapTransformation {
+    private static final String ID="de.danoeh.antennapod.core.glide.FastBlurTransformation";
```

Comment:
Please add spaces before and after the equals sign

Reply:
OK!
I've done this.